plugins/aws/resoto_plugin_aws/__init__.py

```diff
@@ -349,6 +351,28 @@ def current_account_id(profile: Optional[str] = None) -> str:
     return session.client("sts").get_caller_identity().get("Account")  # type: ignore
 
 
+def set_account_name(account: AwsAccount) -> None:
```

Comment:
Consider to move this function inside `set_account_names`

Reply:
Good point! Will do.